packages/types/src/messages/index.ts

```diff
@@ -4,7 +4,7 @@ export { FunctionDefinition, DescribeSequenceMessage, DescribeSequenceMessageDat
 export { StatusMessageData, StatusMessage } from "./status";
 export { ErrorMessage, ErrorMessageData } from "./error";
 export { EventMessage, EventMessageData } from "./event";
-export { HandshakeMessage } from "./handshake";
+export * from "./handshake";
```

Comment:
```suggestion
export * from "./handshake";
```

Let's export named symbols only.

Reply:
fixed